src/typography/src/ListItem.js

```diff
@@ -1,12 +1,70 @@
 import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
```

Comment:
Were these changes to `ListItem` intentional?

Reply:
I am not sure why this is showing up, since this is already in v4.